Unity-Technologies/ScriptableRenderPipeline
# Add support to run post-processing on each camera in the stack

### Checklist for PR maker
- [x ] Have you added a backport label (if needed)? For example, the `need-backport-2019.3` label. After you backport the PR, the label changes to `backported-2019.3`.

---
### Purpose of this PR
 This PR adds support to run post-processing in each camera in the stack, as opposed to run once at the end of the stack. 
 
  This fixes the issue with the previous solution that camera stacking would always apply post-processing on UI screen space and world space UI. 

  Now it's possible to run post-processing in the middle of the stack while still compositing results on top.

---
### Testing status

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] Built a player
- [x ] Checked new UI names with UX convention
- [x ] C# and shader warnings (supress shader cache to see them)
- [x ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 
 Tested several camera stacking demos (cockpit, 3D UI, shooter, shop menu) from Universal Rendering Examples
  Tested BoatAttack for regression.

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.
